### PR TITLE
Add image upload fields for articles

### PR DESCRIPTION
**Pull request by kshepard**

### CODE REVIEW

File: python/cac_tripplanner/cms/models.py [hunk 1]
```diff
@@ -1,9 +1,14 @@
+from ckeditor.fields import RichTextField
```
Comment (flibbertigibbet) [inline python/cac_tripplanner/cms/models.py]: Minor nit:  don't we usually import non-django things below django things?

File: python/cac_tripplanner/destinations/models.py [hunk 2]
```diff
@@ -1,26 +1,13 @@
+from ckeditor.fields import RichTextField
```
Comment (flibbertigibbet) [inline python/cac_tripplanner/destinations/models.py]: Same import ordering here.

**Comment by flibbertigibbet** [comment]
+1

File: python/cac_tripplanner/cms/models.py (hunk 1 above)
Comment (kshepard) [inline python/cac_tripplanner/cms/models.py]: I was reviewing the coding guidelines yesterday and tried to match it to that: "Imports should be grouped into three sections at the top of the file: Python built-in modules, third-party modules, and local source code (stuff in source control)." Maybe that needs to be revised to have a fourth section for django modules? In the meantime I'll switch it as you suggested.

Comment (CloudNiner) [inline python/cac_tripplanner/cms/models.py]: I thought this was handled as:
built-in modules
non-django third-party modules
django modules
django third-party modules
local source

Either way, we should move the discussion off line and clarify whatever we choose in the coding guidelines.

**Pull request merged**